docs/Documentation/Drivers/Image/Params.md

```diff
@@ -1,145 +1,102 @@
 # VIC Model Parameters
 
-!!! Warning
-    Docs not written yet.
+To convert ASCII parameter files that were used in previous versions of VIC, you may follow instructions laid out in [ASCII to NetCDF Parameter Conversion](Ascii_to_NetCDF_params.md).
 
-# Converting from VIC 4 to VIC 5
+We have also put together an example [Ipython notebook](../../../../samples/notebooks/example_reformat_vic4_parameters_to_vic5image.ipynb), that provides the steps necessary to do this conversion.
 
-We are developing a few scripts to aid users in moving from VIC 4 to VIC 5.  These scripts are currently distributed via the [`tonic`](https://github.com/UW-Hydro/tonic) [Python](https://www.python.org/) package.
+After going through these steps and setting up the tonic virtual environment, to look at the parameter file, you can use 'nco' to show the file contents:
```

Comment:
I don't want to give users the idea that the only way to run the image driver is to convert from the classic driver parameters.  So I'm rewriting this section, something like...

The Image Driver uses the (NetCDF)[http://www.unidata.ucar.edu/software/netcdf/] file format for its input model parameters. It is possible to convert the VIC ASCII style parameters to this format.  We have put together an example ([Tutorial](Ascii_to_NetCDF_params.md) and [Ipython notebook](../../../../samples/notebooks/example_reformat_vic4_parameters_to_vic5image.ipynb)), that provides an example of how to do this conversion. Our example uses the  [`tonic`](https://github.com/UW-Hydro/tonic) [Python](https://www.python.org/) package. 

Regardless of the method used to generate the Image Driver parameters, you should end up with a NetCDF file that looks like this:

```
>>>ncdump -h Stehekin_test_params_20160327.FROZEN_SOIL.nc 
netcdf Stehekin_test_params_20160327.FROZEN_SOIL {
dimensions:
	lat = 4 ;
	lon = 5 ;
	nlayer = 3 ;
	snow_band = 5 ;
	veg_class = 12 ;
	string20 = 20 ;
	root_zone = 3 ;
	month = 12 ;
variables:
	double lat(lat) ;
		lat:units = "degrees_north" ;
		lat:long_name = "latitude of grid cell center" ;
	double lon(lon) ;
		lon:units = "degrees_east" ;
		lon:long_name = "longitude of grid cell center" ;
	int mask(lat, lon) ;
		mask:long_name = "area of grid cell" ;
		mask:comment = "0 value indicates cell is not active" ;
		mask:note = "unitless" ;
		mask:standard_name = "area" ;
		mask:units = "m2" ;
		mask:axis = "Y" ;
	int layer(nlayer) ;
		layer:long_name = "soil layer" ;
	int run_cell(lat, lon) ;
		run_cell:_FillValue = -2147483647 ;
		run_cell:units = "N/A" ;
		run_cell:description = "1 = Run Grid Cell, 0 = Do Not Run" ;
		run_cell:long_name = "run_cell" ;
	int gridcell(lat, lon) ;
		gridcell:_FillValue = -2147483647 ;
		gridcell:units = "N/A" ;
		gridcell:description = "Grid cell number" ;
		gridcell:long_name = "gridcell" ;
	double lats(lat, lon) ;
		lats:_FillValue = 9.96920996838687e+36 ;
		lats:units = "degrees" ;
		lats:description = "Latitude of grid cell" ;
		lats:long_name = "lats" ;
	double lons(lat, lon) ;
		lons:_FillValue = 9.96920996838687e+36 ;
		lons:units = "degrees" ;
		lons:description = "Longitude of grid cell" ;
		lons:long_name = "lons" ;
	double infilt(lat, lon) ;
		infilt:_FillValue = 9.96920996838687e+36 ;
		infilt:units = "mm/day" ;
		infilt:description = "Variable infiltration curve parameter (binfilt)" ;
		infilt:long_name = "infilt" ;
	double Ds(lat, lon) ;
		Ds:_FillValue = 9.96920996838687e+36 ;
		Ds:units = "fraction" ;
		Ds:description = "Fraction of Dsmax where non-linear baseflow begins" ;
		Ds:long_name = "Ds" ;
	double Dsmax(lat, lon) ;
		Dsmax:_FillValue = 9.96920996838687e+36 ;
		Dsmax:units = "mm/day" ;
		Dsmax:description = "Maximum velocity of baseflow" ;
		Dsmax:long_name = "Dsmax" ;
	double Ws(lat, lon) ;
		Ws:_FillValue = 9.96920996838687e+36 ;
		Ws:units = "fraction" ;
		Ws:description = "Fraction of maximum soil moisture where non-linear baseflow occurs" ;
		Ws:long_name = "Ws" ;
	double c(lat, lon) ;
		c:_FillValue = 9.96920996838687e+36 ;
		c:units = "N/A" ;
		c:description = "Exponent used in baseflow curve, normally set to 2" ;
		c:long_name = "c" ;
	double expt(nlayer, lat, lon) ;
		expt:_FillValue = 9.96920996838687e+36 ;
		expt:units = "N/A" ;
		expt:description = "Exponent n (=3+2/lambda) in Campbells eqn for hydraulic conductivity, HBH 5.6 (where lambda = soil pore size distribution parameter).  Values should be > 3.0." ;
		expt:long_name = "expt" ;
	double Ksat(nlayer, lat, lon) ;
		Ksat:_FillValue = 9.96920996838687e+36 ;
		Ksat:units = "mm/day" ;
		Ksat:description = "Saturated hydrologic conductivity" ;
		Ksat:long_name = "Ksat" ;
	double phi_s(nlayer, lat, lon) ;
		phi_s:_FillValue = 9.96920996838687e+36 ;
		phi_s:units = "mm/mm" ;
		phi_s:description = "Soil moisture diffusion parameter" ;
		phi_s:long_name = "phi_s" ;
	double init_moist(nlayer, lat, lon) ;
		init_moist:_FillValue = 9.96920996838687e+36 ;
		init_moist:units = "mm" ;
		init_moist:description = "Initial layer moisture content" ;
		init_moist:long_name = "init_moist" ;
	double elev(lat, lon) ;
		elev:_FillValue = 9.96920996838687e+36 ;
		elev:units = "m" ;
		elev:description = "Average elevation of grid cell" ;
		elev:long_name = "elev" ;
	double depth(nlayer, lat, lon) ;
		depth:_FillValue = 9.96920996838687e+36 ;
		depth:units = "m" ;
		depth:description = "Thickness of each soil moisture layer" ;
		depth:long_name = "depth" ;
	double avg_T(lat, lon) ;
		avg_T:_FillValue = 9.96920996838687e+36 ;
		avg_T:units = "C" ;
		avg_T:description = "Average soil temperature, used as the bottom boundary for soil heat flux solutions" ;
		avg_T:long_name = "avg_T" ;
	double dp(lat, lon) ;
		dp:_FillValue = 9.96920996838687e+36 ;
		dp:units = "m" ;
		dp:description = "Soil thermal damping depth (depth at which soil temperature remains constant through the year, ~4 m)" ;
		dp:long_name = "dp" ;
	double bubble(nlayer, lat, lon) ;
		bubble:_FillValue = 9.96920996838687e+36 ;
		bubble:units = "cm" ;
		bubble:description = "Bubbling pressure of soil. Values should be > 0.0" ;
		bubble:long_name = "bubble" ;
	double quartz(nlayer, lat, lon) ;
		quartz:_FillValue = 9.96920996838687e+36 ;
		quartz:units = "fraction" ;
		quartz:description = "Quartz content of soil" ;
		quartz:long_name = "quartz" ;
	double bulk_density(nlayer, lat, lon) ;
		bulk_density:_FillValue = 9.96920996838687e+36 ;
		bulk_density:units = "kg/m3" ;
		bulk_density:description = "Bulk density of soil layer" ;
		bulk_density:long_name = "bulk_density" ;
	double soil_density(nlayer, lat, lon) ;
		soil_density:_FillValue = 9.96920996838687e+36 ;
		soil_density:units = "kg/m3" ;
		soil_density:description = "Soil particle density, normally 2685 kg/m3" ;
		soil_density:long_name = "soil_density" ;
	double off_gmt(lat, lon) ;
		off_gmt:_FillValue = 9.96920996838687e+36 ;
		off_gmt:units = "hours" ;
		off_gmt:description = "Time zone offset from GMT. This parameter determines how VIC interprets sub-daily time steps relative to the model start date and time." ;
		off_gmt:long_name = "off_gmt" ;
	double Wcr_FRACT(nlayer, lat, lon) ;
		Wcr_FRACT:_FillValue = 9.96920996838687e+36 ;
		Wcr_FRACT:units = "fraction" ;
		Wcr_FRACT:description = "Fractional soil moisture content at the critical point (~70%% of field capacity) (fraction of maximum moisture)" ;
		Wcr_FRACT:long_name = "Wcr_FRACT" ;
	double Wpwp_FRACT(nlayer, lat, lon) ;
		Wpwp_FRACT:_FillValue = 9.96920996838687e+36 ;
		Wpwp_FRACT:units = "fraction" ;
		Wpwp_FRACT:description = "Fractional soil moisture content at the wilting point (fraction of maximum moisture)" ;
		Wpwp_FRACT:long_name = "Wpwp_FRACT" ;
	double rough(lat, lon) ;
		rough:_FillValue = 9.96920996838687e+36 ;
		rough:units = "m" ;
		rough:description = "Surface roughness of bare soil" ;
		rough:long_name = "rough" ;
	double snow_rough(lat, lon) ;
		snow_rough:_FillValue = 9.96920996838687e+36 ;
		snow_rough:units = "m" ;
		snow_rough:description = "Surface roughness of snowpack" ;
		snow_rough:long_name = "snow_rough" ;
	double annual_prec(lat, lon) ;
		annual_prec:_FillValue = 9.96920996838687e+36 ;
		annual_prec:units = "mm" ;
		annual_prec:description = "Average annual precipitation." ;
		annual_prec:long_name = "annual_prec" ;
	double resid_moist(nlayer, lat, lon) ;
		resid_moist:_FillValue = 9.96920996838687e+36 ;
		resid_moist:units = "fraction" ;
		resid_moist:description = "Soil moisture layer residual moisture." ;
		resid_moist:long_name = "resid_moist" ;
	int fs_active(lat, lon) ;
		fs_active:_FillValue = -2147483647 ;
		fs_active:units = "binary" ;
		fs_active:description = "If set to 1, then frozen soil algorithm is activated for the grid cell. A 0 indicates that frozen soils are not computed even if soil temperatures fall below 0C." ;
		fs_active:long_name = "fs_active" ;
	int snow_band(snow_band) ;
		snow_band:long_name = "snow band" ;
	double cellnum(lat, lon) ;
		cellnum:_FillValue = 9.96920996838687e+36 ;
		cellnum:units = "N/A" ;
		cellnum:description = "Grid cell number (should match numbers assigned in soil parameter file)" ;
	double AreaFract(snow_band, lat, lon) ;
		AreaFract:_FillValue = 9.96920996838687e+36 ;
		AreaFract:units = "fraction" ;
		AreaFract:description = "Fraction of grid cell covered by each elevation band. Sum of the fractions must equal 1." ;
	double elevation(snow_band, lat, lon) ;
		elevation:_FillValue = 9.96920996838687e+36 ;
		elevation:units = "m" ;
		elevation:description = "Mean (or median) elevation of elevation band. This is used to compute the change in air temperature from the grid cell mean elevation." ;
	double Pfactor(snow_band, lat, lon) ;
		Pfactor:_FillValue = 9.96920996838687e+36 ;
		Pfactor:units = "fraction" ;
		Pfactor:description = "Fraction of cell precipitation thatfalls on each elevation band. Total must equal 1. To ignore effects of elevation on precipitation, set these fractions equal to the area fractions." ;
	int veg_class(veg_class) ;
		veg_class:long_name = "Vegetation Class" ;
	char veg_descr(veg_class, string20) ;
		veg_descr:long_name = "Vegetation Class Description" ;
	int root_zone(root_zone) ;
		root_zone:long_name = "root zone" ;
	int month(month) ;
		month:long_name = "month of year" ;
	int Nveg(lat, lon) ;
		Nveg:_FillValue = -2147483647 ;
		Nveg:long_name = "Nveg" ;
		Nveg:units = "N/A" ;
		Nveg:description = "Number of vegetation tiles in the grid cell" ;
	double Cv(veg_class, lat, lon) ;
		Cv:_FillValue = 9.96920996838687e+36 ;
		Cv:long_name = "Cv" ;
		Cv:units = "fraction" ;
		Cv:description = "Fraction of grid cell covered by vegetation tile" ;
	double root_depth(veg_class, root_zone, lat, lon) ;
		root_depth:_FillValue = 9.96920996838687e+36 ;
		root_depth:long_name = "root_depth" ;
		root_depth:units = "m" ;
		root_depth:description = "Root zone thickness (sum of depths is total depth of root penetration)" ;
	double root_fract(veg_class, root_zone, lat, lon) ;
		root_fract:_FillValue = 9.96920996838687e+36 ;
		root_fract:long_name = "root_fract" ;
		root_fract:units = "fraction" ;
		root_fract:description = "Fraction of root in the current root zone" ;
	double LAI(veg_class, month, lat, lon) ;
		LAI:_FillValue = 9.96920996838687e+36 ;
		LAI:long_name = "LAI" ;
		LAI:units = "m2/m2" ;
		LAI:description = "Leaf Area Index, one per month" ;
	int overstory(veg_class, lat, lon) ;
		overstory:_FillValue = -2147483647 ;
		overstory:long_name = "overstory" ;
		overstory:units = "N/A" ;
		overstory:description = "Flag to indicate whether or not the current vegetation type has an overstory (1 for overstory present [e.g. trees], 0 for overstory not present [e.g. grass])" ;
	double rarc(veg_class, lat, lon) ;
		rarc:_FillValue = 9.96920996838687e+36 ;
		rarc:long_name = "rarc" ;
		rarc:units = "s/m" ;
		rarc:description = "Architectural resistance of vegetation type (~2 s/m)" ;
	double rmin(veg_class, lat, lon) ;
		rmin:_FillValue = 9.96920996838687e+36 ;
		rmin:long_name = "rmin" ;
		rmin:units = "s/m" ;
		rmin:description = "Minimum stomatal resistance of vegetation type (~100 s/m)" ;
	double wind_h(veg_class, lat, lon) ;
		wind_h:_FillValue = 9.96920996838687e+36 ;
		wind_h:long_name = "wind_h" ;
		wind_h:units = "m" ;
		wind_h:description = "Height at which wind speed is measured." ;
	double RGL(veg_class, lat, lon) ;
		RGL:_FillValue = 9.96920996838687e+36 ;
		RGL:long_name = "RGL" ;
		RGL:units = "W/m^2." ;
		RGL:description = "Minimum incoming shortwave radiation at which there will be transpiration. For trees this is about 30 W/m^2, for crops about 100 W/m^2." ;
	double rad_atten(veg_class, lat, lon) ;
		rad_atten:_FillValue = 9.96920996838687e+36 ;
		rad_atten:long_name = "rad_atten" ;
		rad_atten:units = "fraction" ;
		rad_atten:description = "Radiation attenuation factor. Normally set to 0.5, though may need to be adjusted for high latitudes." ;
	double wind_atten(veg_class, lat, lon) ;
		wind_atten:_FillValue = 9.96920996838687e+36 ;
		wind_atten:long_name = "wind_atten" ;
		wind_atten:units = "fraction" ;
		wind_atten:description = "Wind speed attenuation through the overstory. The default value has been 0.5." ;
	double trunk_ratio(veg_class, lat, lon) ;
		trunk_ratio:_FillValue = 9.96920996838687e+36 ;
		trunk_ratio:long_name = "trunk_ratio" ;
		trunk_ratio:units = "fraction" ;
		trunk_ratio:description = "Ratio of total tree height that is trunk (no branches). The default value has been 0.2." ;
	double albedo(veg_class, month, lat, lon) ;
		albedo:_FillValue = 9.96920996838687e+36 ;
		albedo:long_name = "albedo" ;
		albedo:units = "fraction" ;
		albedo:description = "Shortwave albedo for vegetation type" ;
	double veg_rough(veg_class, month, lat, lon) ;
		veg_rough:_FillValue = 9.96920996838687e+36 ;
		veg_rough:long_name = "veg_rough" ;
		veg_rough:units = "m" ;
		veg_rough:description = "Vegetation roughness length (typically 0.123 * vegetation height)" ;
	double displacement(veg_class, month, lat, lon) ;
		displacement:_FillValue = 9.96920996838687e+36 ;
		displacement:long_name = "displacement" ;
		displacement:units = "m" ;
		displacement:description = "Vegetation displacement height (typically 0.67 * vegetation height)" ;

// global attributes:
		:description = "VIC parameter file" ;
		:history = "Created: Mon Mar 28 09:32:46 2016\n",
			"/Users/jhamman/anaconda/lib/python3.4/site-packages/ipykernel/__main__.py -f /Users/jhamman/Library/Jupyter/runtime/kernel-b8627880-fe9e-4d1a-b2d7-fa3fb6a23e22.json\n",
			"" ;
		:source = "/Users/jhamman/anaconda/lib/python3.4/site-packages/ipykernel/__main__.py" ;
		:username = "jhamman" ;
		:host = "Joes-iMac.local" ;
```